src/helpers/tests/artifacts/user2.gen.ts

```diff
@@ -7,6 +7,9 @@
 
 import mongoose from "mongoose";
 
+import { Prop, Schema } from '@nestjs/mongoose';
```

Comment:
Question: what IS the user2 schema used for?

Reply:
`generator.test.ts` has 3 integration tests - device, user, and user2, and I've added report as a 4th. The `<test-name>.gen.ts` files are the expectation of running the corresponding `<test-name>.ts` through the generator.
I don't fully understand what functionality each of the 3 existing tests are focused on. But I've been updating the expectations as I add functionality. Does that answer the question?